service/server.py

```diff
@@ -18,20 +28,72 @@ def healthcheck():
     return "OK"
 
 
+def get_property_address(postcode):
+    client = Elasticsearch([ELASTIC_SEARCH_ENDPOINT])
+    search = Search(
+        using=client, index='landregistry', doc_type='property_by_postcode')
+    query = search.filter('term', postcode=postcode)
+    return query.execute().hits
+
+
+def format_address_into_single_string(address_record):
+    address = [
+        address_record.buildingNumber,
+        address_record.thoroughfareName,
+        address_record.postTown,
+        address_record.postCode
+    ]
+    formatted_address = ", ".join(address)
+    return formatted_address
+
+
+def get_titles_for_uprns(uprns):
+    title_number_uprns = TitleNumbersUprns.query.filter(
+        TitleNumbersUprns.uprn.in_(uprns)).all()
+    title_numbers = map(lambda x: x.title_number, title_number_uprns)
```

Comment:
We could make it a set, so that we would be sure there are no duplicated title numbers.